src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpNoProxy.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Http
+{
+    internal sealed class HttpNoProxy : IWebProxy
+    {
+        public HttpNoProxy() {}
```

Comment:
Nit: this shouldn't be necessary

Reply:
I'll remove it. I need to remember that the compiler is smart.